users/constants.py

```diff
@@ -1,3 +1,5 @@
 """User constants"""
 
 USERNAME_MAX_LEN = 30
+
+USER_REGISTRATION_FAILED_MSG = "Unable to register at this time, please try again later"
```

Comment:
We don't need this message. Can we use the existing general Registration error message? with customer support link

Reply:
Can you please point to the existing general registration error message? The error message is subjective to what we want to use, but I have followed the format used for temporary failures here: https://github.com/mitodl/mitxpro/blob/master/authentication/serializers.py#L177

For further info, please see this comment https://github.com/mitodl/mitxpro/pull/2994#discussion_r1661419126.

Reply:
Well, In that case i think of these two questions:

1. Can we make it a constant and reuse?
2. In case of that error message, Does the registration stop hard or soft? e.g. hard means that the registration page is closed and an error page is shown. Soft means, the user stays on the registration page and sees the error somewhere?

Reply:
> Can we make it a constant and reuse?

Yes, we can make it reusable.

> In case of that error message, Does the registration stop hard or soft? e.g. hard means that the registration page is closed and an error page is shown. Soft means, the user stays on the registration page and sees the error somewhere?

Registration would be soft i.e user stays on the registration page. This is how it will look like:

<img width="1280" alt="notification error" src="https://github.com/mitodl/mitxpro/assets/41048311/dc419205-9a27-4167-a080-e0a184b75647">